### DISCUSSION TITLE
Qwen 2 inference problem

### DISCUSSION THREAD

**Discussion by Sadeghi85** [turboderp/exllamav2#492] [Q&A]
I tried  [ExLlamaV2 of Qwen 2 7b](https://huggingface.co/LoneStriker/Qwen2-7B-Instruct-8.0bpw-h8-exl2), but the output is gibberish. There is a discussion over at llama.cpp: https://github.com/ggerganov/llama.cpp/issues/7805

**Comment by Anthonyg5005** [comment]
Seems to be working fine for me. I'm using chat completions through tabbyAPI.\
ExllamaV2 version: 0.1.4
\
![image](https://github.com/turboderp/exllamav2/assets/43303671/67a96559-e310-4ff3-a4e9-b6e96d69c7c7)
\
![image](https://github.com/turboderp/exllamav2/assets/43303671/ad2ae7ff-c0e2-4c05-9b94-8930256d31ca)
